Makefile

```diff
@@ -15,51 +15,59 @@ else
 	COMPOSER_ARGS := --classmap-authoritative
 	ZIP_NAME := bmlt-root-server-3.0.0-build$(GITHUB_RUN_NUMBER)-$(GITHUB_SHA).zip
 endif
-
 BASE_IMAGE := public.ecr.aws/bmlt/bmlt-root-server-base:latest
+CROUTON_JS := src/legacy/client_interface/html/croutonjs/crouton.js
+VENDOR_AUTOLOAD := src/vendor/autoload.php
+ZIP_FILE := build/bmlt-root-server.zip
+
+$(VENDOR_AUTOLOAD):
+	docker run $(DOCKER_ARGS) -v $(shell pwd)/src:/app -w /app $(BASE_IMAGE) composer install $(COMPOSER_ARGS)
+
+$(CROUTON_JS):
+	curl -sLO https://github.com/bmlt-enabled/crouton/releases/latest/download/croutonjs.zip
+	mkdir -p src/legacy/client_interface/html/croutonjs
+	unzip croutonjs.zip -d src/legacy/client_interface/html/croutonjs
+	rm -f croutonjs.zip
+	rm -f src/legacy/client_interface/html/croutonjs/*.html
+	rm -f src/legacy/client_interface/html/croutonjs/*.json
+	rm -rf src/legacy/client_interface/html/croutonjs/examples
+
+$(ZIP_FILE): $(VENDOR_AUTOLOAD) $(CROUTON_JS)
+	mkdir -p build
+	cp -r src build/main_server
+	cd build && zip -r $(shell basename $(ZIP_FILE)) main_server
+	rm -rf build/main_server
 
-.PHONY: help
 help:  ## Print the help documentation
```

Comment:
can we move this to be the top target. so if you run make with no target or phony it displays the help